client/src/lib/slate-pointers/PointerImportNode.tsx

```diff
@@ -66,6 +68,16 @@ export class PointerImportNode extends React.Component<any, any> {
     }
   };
 
+  public handleClickingClosedPointer = (e, pointerId) => {
+    this.props.openClosedPointer(pointerId);
+    e.stopPropagation();
+  }
+
+  public handleClickingOpenPointer = (e, pointerId) => {
```

Comment:
I'd have used `handleOpenPointerClick` and `handleClosedPointerClick` based on the analogy with `handleClick`, which is much more common than `handleClicking`.